recipe/meta.yaml

```
@@ -1,65 +1,59 @@
 {% set name = "mizani" %}
 {% set version = "0.9.2" %}
-{% set bundle = "tar.gz" %}
-{% set hash = "1d481a4dc673caa9b7cfdc6505b9401f0e9a9f43434d748df0678a1a4017b0e2" %}
-{% set build = 0 %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  sha256: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1d481a4dc673caa9b7cfdc6505b9401f0e9a9f43434d748df0678a1a4017b0e2
 
 build:
-  number: {{ build }}
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.8.0
+    - python
     - pip
     - setuptools
     - setuptools_scm
-
+    - wheel
   run:
-    - python >=3.8.0
-    - numpy >=1.19.0
-    - scipy >=1.5.0
-    - pandas >=1.3.5
+    - python
+    - {{ pin_compatible('numpy') }}
```

```suggestion
    - numpy>=1.19.0
```
pin_compatible is suitable when numpy is set in host.
mizani only uses the python api of numpy, so we can go with what is defined in pyproject.toml